src/Form.svelte

```diff
@@ -179,7 +180,7 @@
     <div>
       {#if field.editable && editable}
         <input name={field.name} class="field" placeholder={field.placeholder}
-               on:change={validate} value={field.value} type="text" {disabled} />
+               on:change={validate} value={field.value || ""} type="text" {disabled} />
```

Comment:
This is to avoid showing an "undefined" as value of an input field, we should prefer to show an empty string which allows us to show the placeholder as default

Reply:
Is this a new issue? Never seen this happen so far :thinking:

Reply:
Yeah it seems that it was introduced somewhere on the way.
Since it currently is not possible to edit ones records. And it seems in part due to passing around undefined values or showing "undefined" as field value

Reply:
Instead of doing it in the `Form.svelte` component, we could also move this to the `registrations/View.svelte` component and define an empty string for values that could be `undefined` but I thought it would be also be applicable for other forms when doing it here